lib/Presentation/Authentication/login_screen.dart

```diff
@@ -75,8 +76,8 @@ class _LoginScreenState extends State<LoginScreen> {
                 ),
               ),
             ),
-            const ButtonWidget(
-              buttonName: AppStrings.signIn,
+             const ButtonWidget(
+              buttonName: AppStrings.signIn, navigatorScreenName: HomeScreen(),
```

Comment:
**question**: why are you navigating to HomeScreen on tap of signIn.
If its kept for temporary then add a TODO